taxi_service/urls.py

```diff
@@ -22,4 +22,8 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    path("registration/", include("django.contrib.auth.urls")),
```

Comment:
I wonder how this path works for you without setting `LOGIN_URL` in settings :thinking: 
Default path for auth urls in django is `accounts/` and it will not resolve default login/logout with a different name unless you specify LOGIN_URL